libs/pika/mpi_base/include/pika/mpi_base/mpi_environment.hpp

```diff
@@ -81,12 +52,12 @@ namespace pika { namespace util {
 
 # include <pika/config/warnings_prefix.hpp>
 
-namespace pika { namespace util {
-        struct PIKA_EXPORT mpi_environment
-        {
-            static bool check_mpi_environment(runtime_configuration const& cfg);
-        };
-}}    // namespace pika::util
+namespace pika::mpi {
+    struct PIKA_EXPORT environment
+    {
+        static bool check_environment(runtime_configuration const& cfg);
+    };
+}    // namespace pika::mpi
```

Comment:
It does look like you should be able to remove the whole `#else` branch here, no? `check_environment` was removed from the other branch, and the namespace was changed to `pika::mpi::detail`, so if it compiles like this it seems like it'll probably compile with this removed as well.

Reply:
Oops, that was a mistake, I meant to remove it all, but must have been distracted when editing ...